CODEOWNERS

```diff
@@ -1 +1 @@
-*   @databricks/field-dev-ecosystem @databricks/eng-dev-ecosystem
+*   @databricks/field-dev-ecosystem @databricks/eng-plat-auto-exp-reviewers
```

Comment:
what is `eng-plat-auto-exp-reviewers`?

Reply:
Just added this: it's currently me, @pietern, and @tanmay-db. It will auto-request reviews from us in a round-robin way so we can keep up with PRs from this repo.